R/plans_helpers.R

```diff
@@ -95,8 +95,9 @@ group_frac <- function(map, group_pop, total_pop = map[[attr(map, "pop_col")]],
 }
 
 
-#' Average a variable by precinct
+#' Average a variable by precinct (Deprecated)
```

Comment:
Can we just alias this instead of deprecating it? The old function name is (1) in use elsewhere and (2) much clearer to people that haven't read your working paper.